game/static/game/css/components.css

```
@@ -0,0 +1,509 @@
+    --card-border: #252545;
+    --card-hover: #1a1a35;
+    --text-primary: #d0d0d0;
+    --text-title: #ffffff;
+    --text-secondary: #aaaaaa;
+    --accent-gold: #f0c040;
+}
+
+/* Localized resets for components to prevent layout breaking on pages without global resets */
+.navbar, .navbar *, .footer, .footer * {
+    box-sizing: border-box;
+    font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)